operations/app/src/environments/tf

```diff
@@ -17,6 +17,17 @@ VERBOSE=${VERBOSE:-0}
 # Where is this script located (and thus, where are the configuration files)?
 SCRIPT_DIR="$(dirname "$(realpath "${0}")")"
 CONFIGS_DIR="${SCRIPT_DIR?}/configurations"
+PLAN_DIR="${SCRIPT_DIR?}/../../plan"
+
+# Does the command require an Azure backend?
+case "${TFCMD?}" in
+validate*)
```

Comment:
`graph` should be here too
also; remove the trailing `*`

Reply:
Agree, remove trailing `*` and add the `graph`.

Reply:
Fixed in next push